src/test/java/complete_task_use_case/CompleteTaskDsInputData.java

```diff
@@ -0,0 +1,5 @@
+package complete_task_use_case;
+
+public class CompleteTaskDsInputData {
+
```

Comment:
If this class is empty do we need it?

Reply:
I litterally do not know how this is here. This is for a different use case and does not appear in my local files. I will try to manually resolve this,